### DISCUSSION TITLE
Is Avalonia.ReactiveUI updated?

### DISCUSSION THREAD

**Discussion by danielmayost** [AvaloniaUI/Avalonia#7667] [Q&A] [answered]
I saw that his repos was last updated only in 2017?
https://github.com/AvaloniaUI/ReactiveUI

**Comment by maxkatz6** [comment, marked as answer]
It's part of the main repo now
https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.ReactiveUI

**Reply by danielmayost** [reply]
Sorry for the misunderstanding.
when i added `"Avalonia.ReactiveUI" Version="0.10.12"` to my project, What version of ReactiveUI is it using, Their updated version?

EDIT: I understood, thank you.

**Comment by kekekeks** [comment]
We were previously shipping a patched build of ReactiveUI to make it compatible with non-hardcoded platforms. Support for 3rd party platform integrations was added to upstream ReactiveUI quite a while ago, so we use packages from nuget.org now.
